src/frequenz/sdk/timeseries/consumer.py

```diff
@@ -0,0 +1,100 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""The logical component for calculating high level consumer metrics for a microgrid."""
+
+import uuid
+
+from frequenz.channels import Sender
+
+from ..actor import ChannelRegistry, ComponentMetricRequest
+from ._quantities import Power
+from .formula_engine import FormulaEngine
+from .formula_engine._formula_engine_pool import FormulaEnginePool
+from .formula_engine._formula_generators import ConsumerPowerFormula
+
+
+class Consumer:
+    """Calculate high level consumer metrics in a microgrid.
+
+    Consumer provides methods for fetching power values from different points
+    in the microgrid. These methods return `FormulaReceiver` objects, which can
+    be used like normal `Receiver`s, but can also be composed to form
+    higher-order formula streams.
```

Comment:
I don't find this docs very useful. I was trying to come up with a description[^1] but the concept of consumer is non trivial, so I thought we should probably add a cross-reference to the Glossary, but then I realized the glossary is also outdated or incomplete, because we have only a definition of [Consumption](https://frequenz-floss.github.io/frequenz-sdk-python/v1.0-dev/user-guide/glossary/#consumption) ( and [Gross Consumption](https://frequenz-floss.github.io/frequenz-sdk-python/v1.0-dev/user-guide/glossary/#gross-consumption)) that refers to the clipped value, and here we actually want to say that this groups everything that is not producing power (in a sense). I notice we are defining a [Load](https://frequenz-floss.github.io/frequenz-sdk-python/v1.0-dev/user-guide/glossary/#load) in the glossary, and to be honest, IMHO this is the most common name for what we call `consumer`, is the part of the circuit we don't know about or control.

There is also the wiki glossary that we can take as reference: https://github.com/frequenz-floss/frequenz-sdk-python/wiki/Glossary#microgrid-power-terminology, for example it says:

> The term consumer was deliberately chosen to indicate that this asset type is expected to predominantly consume electric power. Under normal circumstances there should be no active sources in this asset type and the metrics consumer_{consumption,production}_power would not be used. Alternative terms: load, demand, sink.

I guess if we don't want to get into the whole load/consumer discussion again, we could just add a `Consumer` entry in the glossary and that's it.

@frequenz-floss/python-sdk-team @cwasicki FYI

[^1]: I also wouldn't mention the types returned, like `FormulaReceiver`, I would explain at a higher level what you can get from this (which for now is just power).